README.md

```diff
@@ -1 +1,50 @@
-This is the Videomap capstone project
+# Videomap
+[http://videomap-step-2020.appspot.com/](http://videomap-step-2020.appspot.com/)
+
+<img src="https://user-images.githubusercontent.com/36008213/91629325-ee36b280-e97c-11ea-9ec8-994a088b64a7.png" width=500>
+<img src="https://user-images.githubusercontent.com/55601789/91622207-dfcfa300-e94a-11ea-83a6-dd8b2095e58e.gif" width=500>
+<img src="https://user-images.githubusercontent.com/36008213/90663960-33314b00-e1ff-11ea-9790-fa5883098d59.png" width=500>
```

Comment:
I really like the second gif which acts like a short demo for our project. Can you (or others) replace the other two with gif as well?